src/traces/bar/calc.js

```diff
@@ -11,6 +11,7 @@
 
 var isNumeric = require('fast-isnumeric');
 
+var Lib = require('../../lib');
```

Comment:
you go back and forth between this and `isArrayOrTypedArray = require('../../lib').isArrayOrTypedArray;` - not a big deal but the latter seems marginally preferable to me when you don't need anything else from `Lib`.

Reply:
cleaned up in 9b8382602b0bf0c8e8f436ae24f6701b6684a7a4